### PR TITLE
Task7

### PR DESCRIPTION
**Pull request by stefera**

### CODE REVIEW

**Comment by Harrison365** [comment]
Hey Sam :) 

Just a few things here:
- you need to test that the returned comment has the correct article_id, rather than any(Number)
- you shouldn't need to insert votes and created_at in your SQL string, these should be created automatically.
- you will need a 404 - article not found - test (i.e. article 9000)
- you will need a  400 - invalid article id - test (i.e. banana)
- you will need a 404 - username not found - test (i.e. username: "sam")

Let us know when you want us to take another look 👍 
Thanks :)

**Comment by stefera** [comment]
Thanks Harrison, made those changes now

File: MODELS.js [hunk 1]
```diff
@@ -67,9 +67,36 @@ const fetchCommentsByArticle = (article_id) => {
...
+     (body, author, article_id)
+     VALUES(
+      $1,$2,$3
+     )
+     RETURNING *;
+    `,
+      [body, username, article_id]
+    )
+    .then(({ rows }) => {
+      if (!rows[0]) {
```
Comment (corkesw) [inline MODELS.js:86]: You shouldn't need this custom handling as the absences of a body or username will fall foul of the `NOT NULL` constraint in the db. Try removing it and handling the psql errors instead.

**Review by corkesw — COMMENTED** [review]
Testing look spot on, I've made a small suggestion that you may wish to action but feel free to merge when ready.

File: MODELS.js (hunk 1 above)
Comment (stefera) [reply MODELS.js:86]: Wierdly i am actually catching a would be 200 (for an empty array because article_id doesn't exist), so I've left it in for now.

**Pull request merged**